Makefile

```diff
@@ -0,0 +1,34 @@
+PACKAGE_NAME=geoana
+
+.PHONY: install coverage lint graphs tests docs clean deploy
+
+install:
+	python setup.py install
+
+coverage:
+	nosetests --logging-level=INFO --with-coverage --cover-package=discretize --cover-html
```

Comment:
Some of the makefile still refers to `discretize`

Reply:
ah, thanks!